cfg/pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+---
+repos:
+-
+  hooks:
+  -
+    id: 'hadolint-docker'
+    files: 'rust/Dockerfile'
+  repo: 'https://github.com/hadolint/hadolint'
+  rev: 'v2.12.0'
+-
+  hooks:
+  -
+    args:
+    - '--manifest-path'
+    - 'rust/processor/Cargo.toml'
+    - '--config'
+    - 'build.rustflags=["--cfg", "tokio_unstable"]'
+    - '--'
+    id: 'cargo-check'
+  -
+    args:
```

Comment:
How about `cargo fmt`?

Reply:
I remember that their code wasn't using `cargo fmt` and I didn't want to create too much diff between this and the original. I just checked because of your comment and it seems that it is well formatted now. I added it.